alf/utils/distributions.py

```diff
@@ -0,0 +1,318 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+import torch
+from torch import Tensor
+
+import torch.distributions as td
+
+
+class InverseTransformSampling(object):
```

Comment:
If I understand it correctly, the usage of such interface in `TruncatedDistribution` implies "zero mean" or more strictly the "even function" of the p.d.f. - should we document that here to help when new `*ITS` are added?